src/js/components/atoms/content-link-list/content-link-list.models.ts

```diff
@@ -1,6 +1,6 @@
-import { IContentLink } from 'components/atoms/content-link/content-link.models';
+// import { IContentLink } from 'components/atoms/content-link/content-link.models';
```

Comment:
I would rather remove a line than comment it.
if you want to go back in time we always have git to do so.